internal/cmd/shell_vim.go

```diff
@@ -47,5 +47,9 @@ func (sh vim) escapeKey(str string) string {
 
 // TODO: Make sure this escaping is valid
 func (sh vim) escapeValue(str string) string {
-	return "'" + strings.Replace(str, "'", "''", -1) + "'"
```

Comment:
what happened to the -1? I am not familiar with vimscript

Reply:
And I'm not familiar with Go, but from looking at their docs, the Replacer struct replaces all matches by default.
![image](https://github.com/user-attachments/assets/75164a03-8160-4077-9cdb-7c1f372f713b)

Reply:
thanks